### PR TITLE
Fixing symmetry for right wings

### PR DESCRIPTION
**Pull request by timryanb**
## Purpose
This PR addresses #244. Symmetric wings can now be used with the aerodynamics solver if they lie in the right side of the symmetry plane. Prior to this, OAS only gave correct results for left wings.

## Expected time until merged
a week

## Type of change

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [ ] Maintenance update
- [ ] Other (please describe)

## Testing
This fix can be tested by taking a standard left wing model and flipping the y nodes a shown below:
```
# Flip left wing to lie on right of symmetry plane
mesh[:, :ny, :] = inputs[mesh_name]
mesh[:, ny+1:, :] = inputs[mesh_name][:, ::-1, :]
mesh[:, ny+1:, 1] *= -1.
```

The new flipped surface can be passed to the aerodynamic component and _should_ give a consistent solution to a left wing or full wing model.

## Checklist

- [x] I have run `flake8` and `black` to make sure the code adheres to PEP-8 and is consistently formatted
- [x] I have run unit and regression tests which pass locally with my changes
- [x] I have added new tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation

### CODE REVIEW

**Review by kanekosh — CHANGES_REQUESTED** [review]
Thank you @timryanb! I added a few comments on the new regression tests.

File: openaerostruct/tests/test_simple_rect_right_aero.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                prob.model.connect(name + ".mesh", point_name + ".aero_states." + name + "_def_mesh")
+
+                prob.model.connect(name + ".t_over_c", point_name + "." + name + "_perf." + "t_over_c")
+
+        # Set up the problem
+        prob.setup()
+
+        prob.run_model()
+
+        assert_near_equal(prob["aero_point_0.wing_perf.CD"][0], 0.03487336411850356, 1e-6)
```
[the remark this answers is not in the document]
Comment (timryanb) [reply openaerostruct/tests/test_simple_rect_right_aero.py]: Should be addressed now

File: openaerostruct/tests/test_aero_ground_effect_right.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        prob.model.add_design_var("wing.twist_cp", lower=-10.0, upper=15.0)
+        prob.model.add_constraint(point_name + ".wing_perf.CL", equals=0.5)
+        prob.model.add_objective(point_name + ".wing_perf.CD", scaler=1e4)
+
+        # Set up and run the optimization problem
+        prob.setup()
+
+        prob.run_driver()
+        # docs checkpoint 1
+        assert_near_equal(prob["aero_point_0.wing_perf.CD"][0], 0.033389699871650073, 1e-6)
```
[the remark this answers is not in the document]
Comment (timryanb) [reply openaerostruct/tests/test_aero_ground_effect_right.py]: Should be addressed now

**Review by kanekosh — APPROVED** [review]
Thank you @timryanb! I'll merge this PR after fixing `black` in a separate PR - they recently broke a thrid-party dependency and we need to version up `black` to fix that.

**Comment by nwu63** [comment]
Maybe we also want to bump the version and make a new release?

**Review by kanekosh — APPROVED** [review]

**Pull request merged**